prov/gni/configure.m4

```diff
@@ -102,6 +102,14 @@ dnl looks like we need to get rid of some white space
 
         ])
 
+	AC_ARG_WITH([kdreg], [AS_HELP_STRING([--with-kdreg],
+                                             [Install directory for kdreg headers])])
+
+        if test "$with_kdreg" != "" && test "$with_kdreg" != "no"; then
+		    gni_CPPFLAGS="-I$with_kdreg/include -DHAVE_KDREG $gni_CPPFLAGS"
+        fi
+
+
```

Comment:
We talked about this before, right?  This is not possible until the package is actually available on a system.  Or is there something else I'm missing?